asiana21-DOCSUP-923 (#103)
BayoNet/ClickHouse: docs/en/sql_reference/functions/other_functions.md

Returns a string with the name of the host that this function was performed on. For distributed processing, this is the name of the remote server host, if the function is performed on a remote server.

## getMacro {#getmacro}

> Get the value for an input string from the [macros](https://clickhouse.tech/docs/en/operations/server_configuration_parameters/settings/#macros) section of the configuration file.

Reviewer: ```suggestion
Gets the parameter value from the [macros](../../operations/server_configuration_parameters/settings.md#macros) section of the server configuration file `config.xml`.
```

**Syntax** 

```sql
getMacro(name);
```

**Parameters**

> - `name` — The input macro name. [String](../../sql_reference/data_types/string.md#string).

Reviewer: ```suggestion
- `name` — Name of the parameter from the `macros` section. [String](../../sql_reference/data_types/string.md#string).
```

**Returned value**

> - The value of corresponding [macros](https://clickhouse.tech/docs/en/operations/server_configuration_parameters/settings/#macros) from configuration file on current server.

Reviewer: ```suggestion
- Value of the specified parameter.
```
